lint-python/action.yml

```
@@ -0,0 +1,19 @@
+name: Lint Python
+description: 'Linting for Python projects'
+inputs:
+  exclude_trailing_comma:
```

```suggestion
  exclude_trailing_comma_path:
```

Also here no strong opinion on that, but without `path` sounds like a mask or boolean.

---

https://github.com/moneymeets/moneymeets-composite-actions/pull/4/commits/a173c6c42f5822ad1b428aff6af7a9d6bae64ce9 :)